src/Plane.js

```diff
@@ -0,0 +1,14 @@
+class Plane {
+    constructor(planeID, location = 'In flight') {
+        this.planeID = planeID;
+        this.location = location
+    }
+
+    planeLocation = (airport) => {
+        if (airport.planesAtAirport.includes(this.planeID)) {
+            return this.location = airport.name;
+        }
```

Comment:
If I were to expand on this method, I would need an else statement to change location back to 'In flight' if it wasn't in a list of airport names (I would have to input a list of airports as an array and use for loops)